test/inductor/test_inplacing_pass.py

```diff
@@ -182,6 +187,40 @@ def f(b):
         # Both list inputs failed to reinplace. So we should have emitted clones for them.
         self.assertEqual(post_grad_graphs.count("aten.clone"), 2)
 
+    @parametrize(
+        "factory_op",
+        [
+            subtest(torch.ones_like, name="ones_like"),
+            subtest(torch.empty_like, name="empty_like"),
+        ],
+    )
+    def test_partitioner_recomputes_factory(self, factory_op):
```

Comment:
I personally like `test_perf` tests, since they're "property-based" (i.e. they measure the amount of bytes read and written). It's possible we don't support HOPs yet or something though.

Reply:
I'll add some test_perf tests too